pkg/replica/diff_disk.go

```diff
@@ -222,30 +222,42 @@ func (d *diffDisk) fullReadAt(buf []byte, offset int64) (int, error) {
 
 func (d *diffDisk) read(target byte, buf []byte, startOffset int64, startSector int64, sectors int64) (int, error) {
```

Comment:
Can you help add a doc for explaining the parameters?

Reply:
Added, thanks